google/cloud/iam/CMakeLists.txt

```diff
@@ -19,7 +19,6 @@ set(DOXYGEN_PROJECT_BRIEF "A C++ Client Library for Google Cloud IAM")
 set(DOXYGEN_PROJECT_NUMBER "${PROJECT_VERSION}")
 set(DOXYGEN_EXAMPLE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/samples
                          ${CMAKE_CURRENT_SOURCE_DIR}/quickstart)
-set(DOXYGEN_EXCLUDE_PATTERNS "*/*_test.cc")
 set(DOXYGEN_EXCLUDE_SYMBOLS "internal" "iam_internal" "iam_testing" "examples")
```

Comment:
not introduced in this PR, but: s/examples/samples/

Reply:
This is **symbols** not directories.  There are none (yet), but when we create helper classes in `google/cloud/$product/samples` they are typically called `google::cloud::$product::*examples*`.  And we (at least I) do not want these to be exported as part of the documentation of the library.

Reply:
Ok, thanks for clarifying the point of the changes.

I think [the documentation](https://www.doxygen.nl/manual/config.html#cfg_exclude_symbols) confused me. I guess the note about `*/test/*` only belongs in EXCLUDE_PATTERNS.